src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.js

```diff
@@ -85,6 +96,11 @@ class ActivityMessage extends React.Component<Props, State> {
         ) : (
             <div className="bcs-ActivityMessage">
                 {formatTaggedMessage(commentToDisplay, id, false, getUserProfileUrl)}
+                {isEdited && (
+                    <span className="bcs-ActivityMessage-edited">
+                        <FormattedMessage {...messages.activityMessageEdited} />
+                    </span>
+                )}
                 {translationEnabled ? this.getButton(isTranslation) : null}
```

Comment:
Does this experience still look as expected with the new `(edited)` text?

Reply:
Yes, I think ` (edited)` isn't a problem. I'll double checked, and it's good.
I'm much more worried about this and #3079 
will check with designers after merging this one

Reply:
I agree, it seems like having the `(edited)` flag next to the date would be better.